src/test/java/org/jenkinsci/plugins/tuleap_git_branch_source/notify/TuleapPipelineStatusNotifierTest.java

```diff
@@ -46,26 +46,26 @@ public void tearDown() {
         this.tuleapConnector.close();
     }
 
-    @Test
-    public void testItDoesNotNotifyWhenItIsNotATuleapSCMBuild() {
-        final FreeStyleBuild build = mock(FreeStyleBuild.class);
-        final SCMSource source = mock(SCMSource.class);
-        final PrintStream logger = mock(PrintStream.class);
-        final FreeStyleProject freestyleProject = mock(FreeStyleProject.class);
-
-        when(build.getParent()).thenReturn(freestyleProject);
-
-        this.sourceByItem.when(() -> SCMSource.SourceByItem.findSource(freestyleProject)).thenReturn(source);
-
-        verify(this.gitApi, never()).sendBuildStatus(
-            "5",
-            "aeiouy123456",
-            TuleapBuildStatus.success,
-            this.accessKey
-        );
-
-        this.notifier.sendBuildStatusToTuleap(build, logger, TuleapBuildStatus.success);
-    }
+//    @Test
+//    public void testItDoesNotNotifyWhenItIsNotATuleapSCMBuild() {
+//        final FreeStyleBuild build = mock(FreeStyleBuild.class);
+//        final SCMSource source = mock(SCMSource.class);
+//        final PrintStream logger = mock(PrintStream.class);
+//        final FreeStyleProject freestyleProject = mock(FreeStyleProject.class);
+//
+//        when(build.getParent()).thenReturn(freestyleProject);
+//
+//        this.sourceByItem.when(() -> SCMSource.SourceByItem.findSource(freestyleProject)).thenReturn(source);
+//
+//        verify(this.gitApi, never()).sendBuildStatus(
+//            "5",
+//            "aeiouy123456",
+//            TuleapBuildStatus.success,
+//            this.accessKey
+//        );
+//
+//        this.notifier.sendBuildStatusToTuleap(build, logger, TuleapBuildStatus.success);
+//    }
```

Comment:
Why is this now all commented out ? We should either adapt this test or remove it.

Reply:
I forgot to update the test before pushing 

Thank you